impress-org/givewp
# Fix: Classic form's payment amount summary reflects custom input amounts

Resolves #6650

## Description

When changing the donation payment via the custom amount field (a user typed input), the data-amount on the element does not update until another event on the form occurs. Because the mutation observer is tracking changes to the elements attribute and the attribute does not immediately change the payment amount field is not updating until the following interaction.  I included an event listener for the input to update the payment amount field when changes are committed to the input.

## Affects
Donation Summary Payment Amount

## Visuals

https://user-images.githubusercontent.com/75056371/209046902-9b03b5f8-ee55-4e44-894e-b426990be44b.mov

## Testing Instructions

- View each form type and submit custom amounts and a donation set level amount for each form.
- Verify payment amounts in the donation summary update accordingly.
- See "Visual" for reference.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203615080998862
